05.ls/ls.rb

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'optparse'
+
+MAX_COLUMN = 3
+SPACE_LENGTH = 3
+
+def main
+  files = search_files(ARGV[0])
+  output(files)
+end
+
+def search_files(path)
+  return [path] if FileTest.file?(path.to_s)
+
+  dirpath = ARGV.empty? ? Dir.getwd : path
+  files = Dir.foreach(dirpath).reject do |file|
+    file.start_with?('.')
+  end
+  files.sort
```

Comment:
```suggestion
  Dir.foreach(dirpath).sort.reject do |file|
    file.start_with?('.')
  end
```

でも行けそうな気がしますが、やや高等テクニックなのでこのままでも構いません〜。